### PR TITLE
Connect Floating help button to help pages

### PR DESCRIPTION
**Pull request by jadmnsaad**
https://app.zenhub.com/workspaces/epictrack-63891ea941d309001fa292cf/issues/gh/bcgov/epic.track/2035

Details:
- Add SupportCenterMenuItem
- Add help pages map  json
- Add algorithm to choose best match
- Disabled scroll lock to prevent it from stopping page scroll and subsequently moving when opened in chrome where the scroll bar is always visible

### CODE REVIEW

File: epictrack-web/src/components/AppHelpButton/SupportCenterMenuItem.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+type HelpPageLink = {
+  epicTrackPath: string;
+  helpPage: string;
+};
+
+const SupportCenterMenuItem = () => {
+  const handleClick = () => {
+    const currentPath = window.location.pathname.split("?")[0]; // Get pathname without query params
+    const currentPathSegments = currentPath.split("/"); // Split current path
```
Comment (dinesh-aot) [inline epictrack-web/src/components/AppHelpButton/SupportCenterMenuItem.tsx:20]: we might not get this path parameter for selecting the help page every time. For instance, if we are on work plan for Project Notification, we should be pointing to the help page which talks about Project Notification where the URL would be https://epictrack-dev.apps.gold.devops.gov.bc.ca/work-plan?work_id=58
This way would fit for regular pages.

File: epictrack-web/src/components/AppHelpButton/HelpMenuItem.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { MenuItem, MenuItemProps } from "@mui/material";
+import { Palette } from "styles/theme";
```
Comment (dinesh-aot) [inline epictrack-web/src/components/AppHelpButton/HelpMenuItem.tsx]: The help button moves when you click. Please check if that is already handled in this PR

File: epictrack-web/src/components/AppHelpButton/SupportCenterMenuItem.tsx (hunk 1 above)
Comment (jadmnsaad) [reply epictrack-web/src/components/AppHelpButton/SupportCenterMenuItem.tsx:20]: how would you figure from this https://epictrack-dev.apps.gold.devops.gov.bc.ca/work-plan?work_id=58 that it should be directed to Project Notification help?

File: epictrack-web/src/components/AppHelpButton/HelpMenuItem.tsx (hunk 2 above)
Comment (jadmnsaad) [reply epictrack-web/src/components/AppHelpButton/HelpMenuItem.tsx]: hmm interesting, It's happening in chrome. I use firefox and it's not happening there, I will look into it

**Review by djnunez-aot — APPROVED** [review]

**Pull request merged**